src/rpc/handlers/Ledger.hpp

```diff
@@ -102,9 +105,9 @@ class LedgerHandler {
     spec([[maybe_unused]] uint32_t apiVersion)
     {
         static auto const rpcSpec = RpcSpec{
-            {JS(full), validation::Type<bool>{}, validation::NotSupported{true}},
+            {JS(full), validation::NotSupported{}},
             {JS(full), check::Deprecated{}},
-            {JS(accounts), validation::Type<bool>{}, validation::NotSupported{true}},
+            {JS(accounts), validation::NotSupported{}},
             {JS(accounts), check::Deprecated{}},
             {JS(owner_funds), validation::Type<bool>{}},
             {JS(queue), validation::Type<bool>{}, validation::NotSupported{true}},
```

Comment:
What about `queue`? You removed it from ForwardingProxy but left value here.

Reply:
Thanks for spotting my mistake - i guess i was too enthusiastic about removing stuff :rage4: